packages/server/index.js

```diff
@@ -15,6 +15,10 @@ require('graceful-fs').gracefulify(require('fs'))
 require('@packages/ts/register')
 require('@packages/coffee/register')
 
+if (process.env.CYPRESS_ENV) {
```

Comment:
why this check here without using a value?

Reply:
this is how I validate that we are in Cypress's main Electron process, not some bootstrapping process (since this runs in Node in dev)

i am guessing there is probably a better way to do this, open to suggestions. the net_profiler works the same way.

Reply:
we do `process.env.CY_NET_PROFILE && process.env.CYPRESS_ENV` there